examples/kubernetes/import-snapshot/README.md

```diff
@@ -0,0 +1,91 @@
+# Importing an existing snapshot into the cluster
+
+It is possible to import an existing snapshot that was created out-of-band (i.e., via the DigitalOcean cloud control panel or the API) into a cluster. This guide outlines the necessary steps.
+
+It is based on the [official Kubernetes blog post introducing volume snapshots](https://kubernetes.io/blog/2018/10/09/introducing-volume-snapshot-alpha-for-kubernetes/).
+
+## Prequisites
+
+You need the ID of the snapshot that you want to import. It can be discovered via `doctl compute snapshot list` or the [API equivalent](https://developers.digitalocean.com/documentation/v2/#list-all-volume-snapshots) (but not via the Cloud control panel at this time of writing). For the sake of this guide, we will assume the snapshot ID to be `92b46522-4798-4544-9d33-8b74ea56adb7`.
+
+## Steps
+
+First we need to manually create a `VolumeSnapshotContent` (which can be viewed as the snapshot equivalent of a `PersistentVolume`). It should reference the snapshot ID in the `snapshotHandle` field:
+
+```yaml
+apiVersion: snapshot.storage.k8s.io/v1alpha1
+kind: VolumeSnapshotContent
+metadata:
+  name: snapshotcontent-manual
+spec:
+  csiVolumeSnapshotSource:
+    driver: dobs.csi.digitalocean.com
+    snapshotHandle: 92b46522-4798-4544-9d33-8b74ea56adb7
+  deletionPolicy: Delete
```

Comment:
I wonder if we should recommend `deletionPolicy: Retain` as the default here. I'm sure there are cases where users want the cluster to "own" the snapshot and delete it when they're done with it, but since it was created out-of-band it seems reasonable that many users will also want to delete it out-of-band.

Reply:
Makes total sense to me. I flipped the field to `Retain` and explained the implications of the two field options below.